node-playwright-webkit/Dockerfile

```diff
@@ -38,6 +35,11 @@ RUN npm config --global set update-notifier false
 # Install all required playwright dependencies for webkit
 RUN PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1 npm_config_ignore_scripts=1 npx playwright install-deps webkit
 
+# Cleanup apt caches
+RUN apt-get clean -y && apt-get autoremove -y \
+    && rm -rf /var/lib/apt/lists/* \
+    && rm -rf /src/*.deb
+
```

Comment:
Is this required?

Reply:
It's something we've done in all images to shave off some extra space from the redundant deb packages cached by apt (that said I never tried it with/without it so...maybe it's not? I could try it out and see if theres a size diff)